kde-misc/kdeconnect/kdeconnect-9999.ebuild

```diff
@@ -25,67 +25,60 @@ COMMON_DEPEND="
 	>=app-crypt/qca-2.3.0:2[qt5(+),ssl]
 	dev-libs/glib:2
 	>=dev-libs/wayland-1.15.0
-	>=dev-qt/qtdbus-${QTMIN}:5
-	>=dev-qt/qtdeclarative-${QTMIN}:5
-	>=dev-qt/qtgui-${QTMIN}:5=
-	>=dev-qt/qtmultimedia-${QTMIN}:5
-	>=dev-qt/qtnetwork-${QTMIN}:5
-	>=dev-qt/qtquickcontrols2-${QTMIN}:5
-	>=dev-qt/qtwayland-${QTMIN}:5
-	>=dev-qt/qtwidgets-${QTMIN}:5
-	>=dev-qt/qtx11extras-${QTMIN}:5
-	>=kde-frameworks/kcmutils-${KFMIN}:5
-	>=kde-frameworks/kconfig-${KFMIN}:5
-	>=kde-frameworks/kconfigwidgets-${KFMIN}:5
-	>=kde-frameworks/kcoreaddons-${KFMIN}:5
-	>=kde-frameworks/kdbusaddons-${KFMIN}:5
-	>=kde-frameworks/kguiaddons-${KFMIN}:5
-	>=kde-frameworks/ki18n-${KFMIN}:5
-	>=kde-frameworks/kiconthemes-${KFMIN}:5
-	>=kde-frameworks/kio-${KFMIN}:5
-	>=kde-frameworks/kirigami-${KFMIN}:5
-	>=kde-frameworks/knotifications-${KFMIN}:5
-	>=kde-frameworks/kpeople-${KFMIN}:5
-	>=kde-frameworks/kservice-${KFMIN}:5
-	>=kde-frameworks/kwidgetsaddons-${KFMIN}:5
-	>=kde-frameworks/kwindowsystem-${KFMIN}:5
-	>=kde-frameworks/qqc2-desktop-style-${KFMIN}:5
-	>=kde-frameworks/solid-${KFMIN}:5
-	>=kde-plasma/libplasma-${KFMIN}:5
+	>=dev-qt/qtdeclarative-${QTMIN}:6
+	>=dev-qt/qtbase-${QTMIN}:6[dbus,network,widgets]
+	>=dev-qt/qtmultimedia-${QTMIN}:6
+	>=dev-qt/qtwayland-${QTMIN}:6
+	>=kde-frameworks/kcmutils-${KFMIN}:6
+	>=kde-frameworks/kconfig-${KFMIN}:6
+	>=kde-frameworks/kconfigwidgets-${KFMIN}:6
+	>=kde-frameworks/kcoreaddons-${KFMIN}:6
+	>=kde-frameworks/kdbusaddons-${KFMIN}:6
+	>=kde-frameworks/kguiaddons-${KFMIN}:6
+	>=kde-frameworks/ki18n-${KFMIN}:6
+	>=kde-frameworks/kiconthemes-${KFMIN}:6
+	>=kde-frameworks/kio-${KFMIN}:6
+	>=kde-frameworks/kirigami-${KFMIN}:6
+	>=kde-frameworks/knotifications-${KFMIN}:6
+	>=kde-frameworks/kpeople-${KFMIN}:6
+	>=kde-frameworks/kservice-${KFMIN}:6
+	>=kde-frameworks/kwidgetsaddons-${KFMIN}:6
+	>=kde-frameworks/kwindowsystem-${KFMIN}:6
+	>=kde-frameworks/qqc2-desktop-style-${KFMIN}:6
+	>=kde-frameworks/solid-${KFMIN}:6
+	>=kde-plasma/libplasma-${KFMIN}:6
 	x11-libs/libxkbcommon
-	bluetooth? ( >=dev-qt/qtbluetooth-${QTMIN}:5 )
-	pulseaudio? ( media-libs/pulseaudio-qt:= )
-	telephony? ( >=kde-frameworks/modemmanager-qt-${KFMIN}:5 )
+	bluetooth? ( >=dev-qt/qtconnectivity-${QTMIN}:6[bluetooth] )
+	pulseaudio? ( media-libs/pulseaudio-qt6:= )
+	telephony? ( >=kde-frameworks/modemmanager-qt-${KFMIN}:6 )
 	X? (
 		x11-libs/libfakekey
 		x11-libs/libX11
 		x11-libs/libXtst
 	)
 "
 DEPEND="${COMMON_DEPEND}
+	!!kde-misc/kdeconnect:5
```

Comment:
blockers must be in RDEPEND, hard blocker is unnecessary and needlessly painful to upgraders.